src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class DisabilityBenefitsApp extends React.Component {
```

Comment:
Should keep an eye on this component and remove it if we never used it for more than this.